README.md

```diff
@@ -97,6 +97,7 @@ parameters are used:
 *  **binddn** ... distinguished name (DN) to bind when reading the user’s entry (default is to bind
                   anonymously).
 *  **bindpw** ... credentials to bind with when reading the user’s entry (default is none).
+*  **ssl** ... does the LDAP server require starttls
```

Comment:
You're correct. I added this as an after-thought really.

Maybe something along the lines of "OpenLDAP SSL mechanism (off, on, start_tls)"?